packages/compiler/src/server/serverlib.ts

```diff
@@ -892,6 +913,18 @@ export function createServer(host: ServerHost): Server {
           await host.applyEdit({ changes: { [documentUri]: vsEdits } });
         }
       }
+    } else if (params.command === Commands.OPEN_RULE_DOC && params.arguments) {
+      const docUrl = params.arguments?.[0] as string;
+      if (process.platform === "win32") {
+        // in windows
+        exec(`start ${docUrl}`);
```

Comment:
please fix the security warning. And make sure these command are tested.

Reply:
This will also only work on windows. Vscode is spice an open command that abstract that anyway 

https://github.com/microsoft/vscode-eslint/blob/main/client/src/client.ts#L304-L305